conf/site-production.conf

```diff
@@ -1,14 +1,6 @@
 server {
-        listen          80 default_server;
-        server_name     app.fromdoppler.com;
-
-        return 301 https://$server_name$request_uri;
-}
-server {
-        listen                  443 default_server ssl;
-        server_name             app.fromdoppler.com;
-        ssl_certificate         /run/secrets/site.crt;
-        ssl_certificate_key     /run/secrets/site.key;
+        listen                  80 default_server;
+        server_name             localhost;
```

Comment:
This is only for testing purposes, right?

Reply:
No. It is the default server, virtualhost and other routing responsibilities will be on _doppler-swarm-proxy_:https://github.com/FromDoppler/doppler-swarm#architecture